### ISSUE TITLE
Null reference exception

### ISSUE THREAD

**Issue by samcragg** [fluentassertions/fluentassertions.analyzers#58]
### Description

When including the analyzer in my project I get 44 build warnings about a null reference exception:

```
CSC : warning AD0001: Analyzer 'FluentAssertions.Analyzers.CollectionShouldOnlyHaveUniqueItemsAnalyzer' threw an exception of type 'System.NullReferenceException' with message 'Object reference not set to an instance of an object.'. [C:\...\UnitTests.csproj]
```

### Complete minimal example reproducing the issue

I've narrowed it down to the following code that causes the issue:

```csharp
public class TestClass
{
    private static string StaticResult { get; set; }

    public static void Main()
    {
        StaticResult = nameof(Main);
    }
}
```

### Expected behavior:

No warning is generated by the analyzer during the build

### Actual behavior:

Multiple warnings are generated during the build

### Versions

FluentAssertions Version 5.4.1
FluentAssertions.Analyzers Version 0.11.0
.NET Core 2.0

### Additional Information

I've been able to reproduce this in a unit test so will submit a PR shortly (this issue is so I can reference it in the PR)

**Comment by samcragg** [comment]
I don't think the change has got into the latest NuGet package 🙁 I've restored it but am still getting the warnings; when I decompiled the analyzer the method still looks like before

**Comment by Meir017** [comment]
hmm, not sure exactly how that happened...
anyhow, Just released `0.11.2` with the updated code
